#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add('frame_id',	str_t,		0,	'frame id', 			'')
gen.add('res',		double_t,	0,	'resolution (m/pixel)',	0.000001, 0,0.01)
gen.add('offset_x',	double_t,	0,	'x offset',				0,-2,2)
gen.add('offset_y',	double_t,	0,	'y offset',				0,-2,2)
gen.add('offset_z',	double_t,	0,	'z offset',				0,-2,2)
gen.add('commit',	bool_t, 	0,	'save changes',			False)

exit(gen.generate('projected_interface_builder', "projected_interface", "InterfaceAdjustments"))